specification/servicefabric/resource-manager/Microsoft.ServiceFabric/2016-09-01/servicefabric.json

```diff
@@ -356,6 +395,96 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/environments/{environment}/clusterVersions/{clusterVersion}": {
+      "get": {
+        "tags": [
+          "ClusterVersion"
+        ],
+        "operationId": "ClusterVersions_GetByEnvironment",
+        "description": "Get cluster code versions by environment",
+        "x-ms-examples": {
+          "Get cluster version": {
+            "$ref": "./examples/ClusterVersionsGetByEnvironment_example.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/locationForClusterCodeVersions"
+          },
+          {
+            "$ref": "#/parameters/environment"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/clusterVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Get cluster code versions successfully",
+            "schema": {
+              "$ref": "#/definitions/ClusterCodeVersionsListResult"
```

Comment:
Because methods that return a list must be named starting with `List`: 
  "code": "ListInOperationName",
  "message": "Since operation 'ClusterVersions_Get' response has model definition 'x-ms-pageable', it should be of the form \"*_list*\". Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.",
  "id": "R1003",
  "validationCategory": "SDKViolation",
  "providerNamespace": "Microsoft.ServiceFabric",
  "resourceType": "locations",
  "sources": [
    "file:///home/travis/build/Azure/azure-rest-api-specs/specification/servicefabric/resource-manager/Microsoft.ServiceFabric/2016-09-01/servicefabric.json:448:8 ($.paths[\"/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/clusterVersions/{clusterVersion}\"].get.operationId)"